docs/changehistory/NextVersion.md

```diff
@@ -91,6 +91,31 @@ The following APIs have been promoted to `public`. Public APIs are guaranteed to
 
 All hyper-modeling APIs are now public. [This interactive sample](https://www.itwinjs.org/sample-showcase/?group=Viewer&sample=hypermodeling-sample&imodel=House+Sample) demonstrates how to use hyper-modeling features.
 
+### [@bentley/imodeljs-frontend](https://www.itwinjs.org/reference/imodeljs-frontend/)
+
+* [ViewGlobeSatelliteTool]($frontend) for viewing the current camera location from approximately a satellite's height above the background map.
```

Comment:
Collapse these into one bullet point describing them all as tools for viewing the iModel in a global context.

Reply:
done

Reply:
This has been pushed to https://github.com/imodeljs/imodeljs/pull/1096